net.rosebirch.nextgroove.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id" : "net.rosebirch.nextgroove",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "nextgroove",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--own-name=org.mpris.MediaPlayer2.nextgroove"
```

Comment:
can you make this follow the flatpak-id instead of nextgroove in the last part? Then this permission can be dropped.